nav2_rviz_plugins/CMakeLists.txt

```diff
@@ -37,6 +37,8 @@ set(nav2_rviz_plugins_headers_to_moc
   include/nav2_rviz_plugins/goal_common
   include/nav2_rviz_plugins/goal_tool.hpp
   include/nav2_rviz_plugins/nav2_panel.hpp
+  include/nav2_rviz_plugins/flat_weighted_arrows_array.hpp
```

Comment:
Can you put this in a separate directory `particle_filter_plugin` or something to keep them organized?